### PR TITLE
feat: v3 API surface

### PR DESCRIPTION
**Pull request by bcoe**
* updates samples to reference `v3` API, rather than `v3beta`.
* exports `v3` API surface.

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "mocha --recursive --timeout 90000",
-    "test-v3": "mocha ./test/v3beta1/*.js"
+    "test-v3": "mocha ./test/v3/*.js"
```
Comment (JustinBeckwith) [inline samples/package.json]: This is weird.  Are these getting run as part of `npm test`?

File: samples/test/v3/.eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  no-warning-comments: off
```
Comment (JustinBeckwith) [inline samples/test/v3/.eslintrc.yml]: We shouldn't need this file - I think this matches what's in the next level up

File: src/v3/.eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  node/no-missing-require: off
```
Comment (JustinBeckwith) [inline src/v3/.eslintrc.yml]: I don't think we want to enable this :) Let's just delete this file

**Comment by nnegrey** [comment]
Just adding myself as a participant so I get notified when this merges to make sure missing samples get added.

**Pull request closed without merging**